### ISSUE TITLE
Server side error: "ReferenceError: File is not defined"

### ISSUE THREAD

**Issue by jordie23** [rpldy/react-uploady#106]
**Describe the bug**
When using Uploady in a SSR app, it throws this error on the server side. `"ReferenceError: File is not defined"`

**To Reproduce**
- Install NextJS (I assume just using this next create app would be the quickest way https://nextjs.org/docs/api-reference/create-next-app ). 
- Install `npm i @rpldy/uploady`
- Create a file in next `pages/test.js` and use the code provided below
- Start dev server, navigate to the /test page

**Expected behavior**
Does not throw an error.

**Versions**
I'm still using version 9.5.2 though of NextJS, their latest release is 10.0.3 and latest on the 9.5.* branch is 9.5.5. Have not had the chance to update yet.

```
"@rpldy/uploady": "^0.8.0",
"react": "16.13.1",
"react-dom": "16.13.1",
"next": "9.5.2",
```

**Code**

This is what I put in the `test.js` file:

```jsx
import Uploady from '@rpldy/uploady'

export default function TestPage() {
  return (
      <Uploady destination={{ url: '/path/to/upload' }}>
        Hello
      </Uploady>
  )
}
```

Stack Trace I get:

```
ReferenceError: File is not defined
    at isNativeFile (<working dir>/node_modules/@rpldy/simple-state/lib/cjs/utils.js:20:25)
    at isProxiable (<working dir>/node_modules/@rpldy/simple-state/lib/cjs/utils.js:24:68)
    at deepProxy (<working dir>/node_modules/@rpldy/simple-state/lib/cjs/createState.js:40:30)
    at _default (<working dir>/node_modules/@rpldy/simple-state/lib/cjs/createState.js:112:46)
    at createUploaderQueue (<working dir>/node_modules/@rpldy/uploader/lib/cjs/queue/uploaderQueue.js:41:50)
    at _default (<working dir>/node_modules/@rpldy/uploader/lib/cjs/processor.js:18:37)
    at _default (<working dir>/node_modules/@rpldy/uploader/lib/cjs/uploader.js:166:45)
    at <working dir>/node_modules/@rpldy/shared-ui/lib/cjs/useUploader.js:33:37
    at Object.useMemo (<working dir>/node_modules/react-dom/cjs/react-dom-server.node.development.js:1239:19)
    at useMemo (<working dir>/node_modules/react/cjs/react.development.js:1521:21)
```

And just tagging the prior comments on this issue for anyone that might be googling: https://github.com/rpldy/react-uploady/issues/76

**Comment by yoavniran** [comment]
thanks for the details @jordie23 
should be fixed now in 0.8.1

**Comment by jordie23** [comment]
Amazing, thanks mate!
